### DISCUSSION TITLE
How to increase the window size?

### DISCUSSION THREAD

**Discussion by grinchee** [jlesage/docker-putty#15] [Q&A] [answered]
I am talking about the second windows after you type in the IP and click open, the black interactive window. 

The other question is the black window appears blank unless you type in the username and password

Is there a way to open multiple sessions at the same time?

Thanks, Great work!

**Comment by jlesage** [comment, marked as answer]
> I am talking about the second windows after you type in the IP and click open, the black interactive window.

The window size should automatically resize to the size of the browser's window.

If you are using a VNC client, you can adjust the size via `DISPLAY_WIDTH` and `DISPLAY_HEIGHT` environment variables.

> The other question is the black window appears blank unless you type in the username and password

Did you try the latest version ? A display glitch has been fixed.

> Is there a way to open multiple sessions at the same time?

PuTTY itself doesn't support multiple sessions.

**Comment by grinchee** [comment]
Thanks. yes the new image fixed the problems.

Great work!

R
Shaoyan
